utils/motorcontroller/defaultconfig/NeoDefaults.java

```diff
@@ -0,0 +1,27 @@
+package frc.team670.mustanglib.utils.motorcontroller.defaultconfig;
+
+import frc.team670.mustanglib.utils.motorcontroller.MotorConfig.SparkMAXFactory.Config;
+
+public class NeoDefaults {
+    // for velocity control, all we need is frame 1, which has the velocity value
```

Comment:
I like to include a link to the source of this information so that someone can go look at it down the road as needed